### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by dfar-io**
Creates README.md information to address #242.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,230 +25,53 @@ Freshli reads dependency information from special files called "dependency manif
...
+dotnet add package Freshli.Core
 ```
 
-## Running `eclint` via docker-compose
+### Alpha Releases
 
-The `eclint` container provides a convenient way to run the `eclint` command
-that's used by GitHub Actions as part of every pull request. To do so simply
-run the following.
+To use an alpha release, you'll need to [set up a GitHub personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) and then create a `nuget.config` file in your project root with the following contents:
```
Comment (mrbiggred) [inline README.md:51]: Will a personal access token be needed once the repo is switched back to public?

File: README.md [hunk 2]
```diff
@@ -25,230 +25,53 @@ Freshli reads dependency information from special files called "dependency manif
...
+## Using Freshli in a Project
 
-### Running `freshli`
+To use Freshli in a project, you have two options:
 
-To run `freshli` from the command line, the project [freshli-cli](https://github.com/corgibytes/freshli-cli) is required.
-Please reference the [README](https://github.com/corgibytes/freshli-cli#getting-started-with-freshli-cli) instructions found there.
+### Stable and Beta Releases
 
+Stable and beta releases are found at https://www.nuget.org/packages/Freshli/ and can be added with the [`dotnet add package`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package) command:
```
Comment (mrbiggred) [inline README.md:43]: No point in adding this to the ReadMe until the package exists.  Let's try for a beta in  [v0.4.0](https://github.com/corgibytes/freshli/milestone/5).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
+## Contributing to `freshli`
```
Comment (mrbiggred) [inline CONTRIBUTING.md:1]: I like the idea of having a Contributing file for how to setup your Dev environment, building, branching, etc, and leaving the ReadMe for consumers of the NuGet package.

**Review by mrbiggred — CHANGES_REQUESTED** [review]
Thanks for breaking out the contribution section to it's own file.  Myself and @dan-hein will have to do some updates before we merge our PRs with ReadMe updates.

I have a question if consumers need a GitHub token to access the alpha NuGet package?  My guess is no once we change the repo back to public but I'm not sure.

My only change request is to remove the reference to the beta version of the package?  We will include that in v0.4.0.

File: README.md (hunk 1 above)
Comment (dfar-io) [reply README.md:51]: Yes - a PAT is always required for Github Packages - https://docs.github.com/en/packages/learn-github-packages/about-github-packages#authenticating-to-github-packages

File: CONTRIBUTING.md (hunk 3 above)
Comment (dfar-io) [reply CONTRIBUTING.md:1]: Removed this section.

**Pull request merged**